packages/components-js/tests/vrt/playwright/helpers/popover-helper.ts

```diff
@@ -8,3 +8,15 @@ export const openAllPopover = async (page: Page): Promise<void> => {
     });
   });
 };
+
+export const setAllPopoverManual = async (page: Page): Promise<void> => {
```

Comment:
Perhaps we can find a better name since it is hard to tell what this really means.